pennylane/ops/__init__.py

```diff
@@ -23,6 +23,7 @@
 from .cv import *
 from .qubit import *
 from .channel import *
+from .op_math import *
```

Comment:
Should we import all, or just the user-facing methods?

If we are importing all, we can get rid of the `from .op_math import adjoint` line above.

Reply:
I think just user facing makes most sense, I will change this